test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageCompatibiltyValidationTests.cs

```diff
@@ -0,0 +1,215 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NuGet.Common;
+using NuGet.Commands;
+using NuGet.Test.Utility;
+using Xunit;
+using NuGet.Packaging.Rules;
+
+namespace NuGet.Packaging.Test
+{
+    public class PackageCompatibiltyValidationTests
```

Comment:
normally unit test files use the same class name as the class it's testing, with the word `Tests` appended on the end. Eg a class testing the `Whatever` class is called `WhateverTests`.